docs/extensions/compose.md

```diff
@@ -294,12 +294,39 @@ The default `HorizontalChildPanelsLayout` layout places child components (panels
 - If the `mode` is `DUAL`, the Main panel is always displayed on the left side, and then the Details and the Extra panels are displayed in a stack on the right side (next to the Main panel).
 - If the `mode` is `TRIPLE`, all panels are displayed horizontally side by side.
 
+You can use window size classes from the `material3-window-size-class` package to determine which `ChildPanelsMode` should be used.
+
+```kotlin title="WindowSizeClass example"
+import androidx.compose.material3.windowsizeclass.WindowWidthSizeClass
+import androidx.compose.material3.windowsizeclass.calculateWindowSizeClass
+
+@Composable
+fun MyApp() {
+    val windowSizeClass = calculateWindowSizeClass()
+    val mode = if (windowSizeClass.widthSizeClass < WindowWidthSizeClass.Expanded) {
```

Comment:
Perhaps we could use `derivedStateOf`?

Reply:
Unfortunately, `derivedStateOf` won't help because there is no compose state here